### ISSUE TITLE
userTiming: Lack of User Timing API should not impact score

### ISSUE THREAD

**Issue by jasonbarry** [sitespeedio/coach#148]
<img width="665" alt="screen shot 2016-07-12 at 11 32 07 am" src="https://cloud.githubusercontent.com/assets/236451/16778774/4bd61bdc-4824-11e6-9161-6ab99cfa31c4.png">

I use webcoach to score my company's production site. We use [React's performance tools](https://facebook.github.io/react/docs/perf.html) for measuring metrics. Those performance measuring calls are dev only and don't get shipped to production. My webcoach score shouldn't be dinged because I'm not using a certain API, one which IMO shouldn't be used in production anyway. 

I'm on the boat where the userTiming check should be removed entirely.

**Comment by soulgalore** [comment]
yeah, thought about if it should be in the info part from the coach or an actual advice when I added it the first time, but I kind of like the idea of that it helps people opening the eyes for the User Timing API, since it cool (but of course can be better in the future) and gives you more opportunities to collect your own RUM data.
